webapp/lib/controller.dart

```diff
@@ -747,6 +781,7 @@ void command(UIAction action, Data data) {
       activeConversation = null;
       view.conversationListPanelView.clearConversationList();
       view.conversationPanelView.clear();
+      view.replyPanelView.noteText = '';
```

Comment:
Can I just check this change was intended? We don't use this field much so far, so would like to make sure that it's not a stray bug that's going to overwrite the field?

Reply:
Yes, this was intended, and it won't overwrite the field, it will only clear the text of the notes HTML element.

What happened was that the notes field wasn't being reset when switching between conversation lists. Most of the times this isn't an issue, as the notes view gets set when a new conversation is selected, and that's by default the first one in the list after a switch. The only time this would be an issue is when switching from one of the lists to the default of no list selected (the "Select the conversation list to be displayed" in the dropdown). 

Hope this makes sense? And sorry, I came across this small bug during testing and so I just fixed it in place - it would have been clearer to do this in a separate PR.